modules/installation-vsphere-machines.adoc

```diff
@@ -17,6 +17,38 @@ Before you install a cluster that contains user-provisioned infrastructure on VM
 
 .Procedure
 
+. Locate the bootstrap Ignition config file `<installation_directory>/bootstrap.ign` that the installation program created.
++
+The bootstrap Ignition config file is too large to fit in a vApp property. Unless you store the file in the extra configuration properties of the virtual machine (VM), you must host it on an external HTTP server.
+
+. Upload the file to your HTTP server.  Note the URL of this file.
+
+. Save the following secondary Ignition config file for your bootstrap node to your computer as `<installation_directory>/merge-bootstrap.ign`.
++
+----
+{
+  "ignition": {
+    "config": {
+      "merge": [
```

Comment:
Calling out here that I changed the filename from what we have in 4.5 docs to reflect `Ignition spec 3.1.0`: 
`"append":  [` --> `"merge":  [`